### PR TITLE
feat(FOROME-1388): add default preset

### PR DESCRIPTION
**Pull request by Lapk1n**
https://quantori.atlassian.net/browse/FOROME-1388

### CODE REVIEW

**Review by EvgeniySobolev — APPROVED** [review]

**Review by QSdmitrioul — APPROVED** [review]

**Review by AndreyUstyumenko — CHANGES_REQUESTED** [review]
After reloading the page, preset switches to List, however Preset1 is selected in the menu
Screen Shot 2022-07-22 at 16 41 59

File: src/pages/ws/ui/variant-drawer/variant-drawer.data.ts [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+
+export const defaultPresetsNames = ['List', 'Preset 1']
+
+export const predefinedPresets: IVariantDrawerGridPreset[] = [
+  {
+    name: 'List',
+    predefinedName: VariantDrawerPredefinedPresets.List,
+  },
+  {
+    name: 'Preset 1',
```
Comment (AndreyUstyumenko) [inline src/pages/ws/ui/variant-drawer/variant-drawer.data.ts]: Let's call it more fancy. For example, Cards View

File: src/pages/ws/ui/variant-drawer/variant-drawer.store.ts [hunk 2]
```diff
@@ -89,6 +81,9 @@ class VariantDrawerStore {
           case VariantDrawerPredefinedPresets.List:
             this.currentGridLayout = []
             break
+          case VariantDrawerPredefinedPresets.Preset1:
+            this.currentGridLayout = preset.layout
+            break
         }
       } else if (preset.layout) {
         this.currentGridLayout = cloneDeep(preset.layout)
```
Comment (AndreyUstyumenko) [inline src/pages/ws/ui/variant-drawer/variant-drawer.store.ts]: maybe just `this.currentGridLayout = preset.layout?.length ? cloneDeep(preset.layout) : []` without ifs and the switch case block?

**Comment by AndreyUstyumenko** [comment]
Preset is still getting reset when reloading. I think it should keep memorizing it across page reloads, it's pretty annoying.

**Review by AndreyUstyumenko — CHANGES_REQUESTED** [review]
Preset is still getting reset after reloading. I think it should keep it across page reloads, it's pretty annoying.

**Review by AndreyUstyumenko — APPROVED** [review]

**Pull request merged**